tests/unit_tests/commandline/test_geoips_get_package.py

```diff
@@ -19,14 +19,17 @@ def command_combinations(self):
         """
         if not hasattr(self, "_cmd_list"):
             self._cmd_list = []
-            base_args = self._get_package_args
+            base_args = ["geoips", "get"]
             # add arguments for retrieving each package
             for pkg_name in self.plugin_package_names:
-                self._cmd_list.append(base_args + [pkg_name])
+                self._cmd_list.append(base_args + ["pkg", pkg_name])
```

Comment:
These aliases are not included in the alias mapping? Or this is explicitly being set for tests?

Reply:
These are included in alias mapping. But are listed explicitly here. I've changed the code to make this dynamic.